<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serveur</class>
 <widget class="QWidget" name="Serveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLineEdit" name="lineEditMeteo"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Liste des messages en cours de diffusion : </string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonSupprimer">
     <property name="text">
      <string>Supprimer un message</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Message d'information :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bulletin météo : </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButtonAlerte">
     <property name="text">
      <string>Créer une alerte </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Message d'alerte : </string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QPushButton" name="pushButtonDiffusion">
     <property name="text">
      <string>Lancer la diffusion</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QPushButton" name="pushButtonMeteo">
     <property name="text">
      <string>Modifier le bulletin météo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="textEditAlerte"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="textEditInformation"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonAjouter">
     <property name="text">
      <string>Ajouter un message </string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="4">
    <widget class="QListWidget" name="listWidgetMessage"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
